README.md

```diff
@@ -356,18 +372,22 @@ Where to discover new R-esources.
 * [R Bloggers](http://www.r-bloggers.com/) - There are people scattered across the Web who blog about R. This is simply an aggregator of many of those feeds.
 * [DataCamp](https://www.datacamp.com/) - Learn R data analytics online.
 * [Quick-R](http://www.statmethods.net/) - An excellent quick reference.
-* [Advanced R](http://adv-r.had.co.nz/) -  An in-progress book site for Advanced R.
+* [Advanced R](http://adv-r.had.co.nz/) - An in-progress book site for Advanced R.
 * [CRAN Task Views](http://cran.r-project.org/web/views/) - Task Views for CRAN packages.
 * [The R Programming Wikibook](https://en.wikibooks.org/wiki/R_Programming) - A collaborative handbook for R. 
 * [R-users](https://www.r-users.com/) - A job board for R users (and the people who are looking to hire them)
+* [R Cookbook](http://www.cookbook-r.com/) - A problem-oriented website that supports the _[R Graphics Cookbook](http://shop.oreilly.com/product/0636920023135.do)_.
 
 ## Books
 
-* [The Art of R Programming](http://shop.oreilly.com/product/9781593273842.do) -  It's a good resource for systematically learning fundamentals such as types of objects, control statements, variable scope, classes and debugging in R.
-* [R in Action](http://www.manning.com/kabacoff2/) - This book aims at all levels of users, with sections for beginning, intermediate and advanced R ranging from "Exploring R data structures" to running regressions and conducting factor analyses.
-* [Use R!](http://www.springer.com/series/6991?detailsPage=titles) - This series of inexpensive and focused books from Springer publish shorter books aimed at practitioners. Books can discuss the use of R in a particular subject area, such as bayesian networks, ggplot2 and Rcpp.
+* _[The Art of R Programming](http://shop.oreilly.com/product/9781593273842.do)_ - It's a good resource for systematically learning fundamentals such as types of objects, control statements, variable scope, classes and debugging in R.
+* _[R Cookbook](http://shop.oreilly.com/product/9780596809164.do)_ - A quick and simple introduction to conducting many common statistical tasks with R.
+* _[Exploratory Data Analysis with R](https://leanpub.com/exdata)_, _[R Programming for Data Science](https://leanpub.com/rprogramming)_ and _[Report Writing for Data Science in R](https://leanpub.com/reportwriting)_ - Books written as part of the Johns Hopkins Data Science Specialization.

```

Comment:
First onw looks nice =)

there is also the new book: [The Art of Data Science](https://leanpub.com/artofdatascience)
which is amazing.
;)